make/Makefile.curl

```diff
@@ -8,7 +8,7 @@ LIBCURL_OPTS =
 #    LIBCURL_OPTS += --with-darwinssl
 #else
     ifneq (,$(findstring mingw,$(BUILD)))
-        LIBCURL_OPTS += --with-ssl --with-winssl
+        LIBCURL_OPTS += --without-ssl --with-winssl
```

Comment:
This config line (which really means to use OpenSSL) was in error before, it only worked accidentally but failed on the latest curl before this update.